src/pages/profile/ProfilePage.tsx

```diff
@@ -45,6 +46,10 @@ export default function ProfilePage() {
               </IonButtons>
 
               <IonTitle>{handle}</IonTitle>
+
+              <IonButtons slot="end">
+                <ProfilePageActions />
```

Comment:
You will also need to add this to `<ProfilePage>` (if not `isSelf`).

ProfilePage = `/profile`
UserPage = `/profile/lemmy.world/u/aeharding` (for example) - you can also access your user data here

Reply:
I think this already works as expected, or at least it works in a way that makes sense:

Go to `/profile` -> "more actions" menu shows link to user modlog
Go to `/profile/[instance]/u/[my_username]` -> no "more actions" menu 
Go to `/profile/[instance]/u/[other_username]` -> "more actions" menu shows links to send message and block user

Unless you also want the modlog link to show in the second scenario. But I don't think that's necessary, since it can already be accessed from the profile.

Reply:
Yes, the ellipse should always show if you're viewing your own profile. To the user their the same page. The code should probably not be duplicated here but thats beyond the scope of this ticket.

Reply:
Okay, gotcha. I'll get it done.